src/ui/components/SecondaryToolbox/react-devtools-inline.d.ts

```diff
@@ -187,3 +187,7 @@ https: declare module "react-devtools-inline/frontend" {
     bridgeAndStore: { bridge?: FrontentBridge; store?: Store }
   ): typeof ReactDevTools;
 }
+
+https: declare module "react-devtools-inline_4_17_0/frontend" {
+  export * from "@types/react-devtools-inline/frontend";
```

Comment:
I am fairly new to TypeScript. Is this a valid way to tell TypeScript that these two modules have the same API?